pages/community/precice-workshop-2025.md

```diff
@@ -108,6 +108,10 @@ Preliminary schedule:
 
 A few weeks after the workshop, we will also have an online follow-up user support session (TBA).
 
+## Travel awards
+
+We have up to 5 travel awards available for those, who cannot join otherwise. Each award includes co-funding of 500€ for travel and accomodation and a waiver for the registration fee. If you are interested in this support, please *briefly* explain your situation by mailing to `precice` at `hsu-hh.de`.
```

Comment:
Why the emphasis on "briefly"? Sounds strange to me. Also, a typo fix.
```suggestion
We have up to 5 travel awards available for those who cannot join otherwise. Each award includes co-funding of 500€ for travel and accomodation and a waiver for the registration fee. If you are interested in this support, please briefly explain your situation by mailing to `precice` at `hsu-hh.de`.
```